src/content/apis/sports/LiveScore API.md

```diff
@@ -0,0 +1,10 @@
+---
+name: LiveScore API
+description: This API helps to query for football, cricket, basketball, tennis, hockey matches, leagues, news, etc… to create a sporting site/application such as livescore.com
+categories:
+  - sports
+features:
+  https: true
+url: https://livescore6.p.rapidapi.com/matches/v2/list-live
```

Comment:
Does the API have any proper documentation?
The current URL you shared doesn't have any proper info about how to use it.